### PR TITLE
Add Key phrase functionality

### PR DESCRIPTION
**Pull request by seastwood**
This is the working skeleton for the user to add a key phrase

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -37,6 +37,9 @@ dependencies {
     implementation 'androidx.appcompat:appcompat:1.5.1'
     implementation 'com.google.android.material:material:1.6.1'
     implementation 'androidx.constraintlayout:constraintlayout:2.1.4'
+    //noinspection GradlePath
+    implementation files('C:/Program Files (x86)/MySQL/Connector J 8.0/mysql-connector-java-8.0.30.jar')
+    implementation 'androidx.test.ext:junit-ktx:1.1.3'
```
Comment (kouYang2015) [inline app/build.gradle]: Might not need anymore. Delete
```suggestion

```

File: app/src/main/java/com/example/vre_app/MainActivity.kt [hunk 2]
```diff
@@ -5,11 +5,23 @@ import android.os.Bundle
...
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
 
+        val goToKeyPhraseMenu = findViewById<Button>(R.id.goToKeyPhraseMenu)
+
+        goToKeyPhraseMenu.setOnClickListener {
+            val intent = Intent(this, KeyWordsActivity::class.java)
+            // start your next activity
+            startActivity(intent)
```
Comment (kouYang2015) [inline app/src/main/java/com/example/vre_app/MainActivity.kt]: Might not need.
```suggestion

```

**Review by kouYang2015 — CHANGES_REQUESTED** [review]
Check comments

File: app/src/main/java/com/example/vre_app/MainActivity.kt (hunk 2 above)
Comment (seastwood) [reply app/src/main/java/com/example/vre_app/MainActivity.kt]: removed in android studio

**Review by kouYang2015 — APPROVED** [review]
LGTM

**Pull request merged**